apps/chat_server/lib/chat_server/contexts/rooms/update_room.ex

```diff
@@ -9,14 +10,15 @@ defmodule ChatServer.UpdateState do
     defstruct [:room]
   end
 
-  def call(params \\ %{}, _user, state) do
-    Logger.info "Updating room " <> inspect(params)
+  def call(room_id, _user, state) do
+    Logger.info "Updating room " <> inspect(room_id)
 
     # TODO verify user is allowed to update room of that type
 
-    with {:ok, record} <- update(params, state),
-         :ok <- broadcast_update(record) do
-      {:ok, record}
+    with {:ok, record} <- GetRoom(room_id),
+         {:ok, record} <- update(record, state),
+          :ok <- broadcast_update(record) do
```

Comment:
The Elixir syntax standard we follow calls for `with` statement clauses to always be indented 5 spaces. Let's drop the additional space on this line.